x-pack/test/functional/page_objects/graph_page.ts

```diff
@@ -38,6 +39,11 @@ export function GraphPageProvider({ getService, getPageObjects }: FtrProviderCon
       );
     }
 
+    async getGraphCircleText() {
+      const chartTypes = await find.allByCssSelector('text.nodeSvgText');
+      return Promise.all(chartTypes.map(getCircleText));
+    }
+
```

Comment:
Not sure what this change is for.  Can it be in a separate PR?  I don't think we're running a graph test in integration-test